javascript-job-search-mvp/index.html

```diff
@@ -75,7 +75,11 @@
 						<li><a href="https://vetswhocode.io/apply"><span class="link-name">Apply</span></a></li>
 						<li><a href="https://vetswhocode.io/contact"><span class="link-name">Contact</span></a></li>
 						<li><a href="https://vetswhocode.io/blog"><span class="link-name">Blog</span></a></li>
+<<<<<<< HEAD
```

Comment:
this was cut and pasted from your last PR. the <<<< HEAD doesn't need to be there, the ====== doesn't need to be there, and the  >>>>> main doesn't need to be there.